server/routes/trees/trees-router.js

```diff
@@ -8,15 +8,16 @@ const {
 const treeHistory = require('../treehistory/treehistory-queries');
 const { createTree, findTreeById, updateTreeById } = require('./trees-queries');
 const { validatePostTree } = require('./trees-validations');
+const { IDForTree } = require('./id');
```

Comment:
A more consistent name might be `createIdForTree()`, or `createTreeIdFromProperties()`.